internal/config/config.go

```diff
@@ -81,7 +82,32 @@ type MTLS struct {
 	TLSSecretNamespace string `mapstructure:"secretNamespace"`
 }
 
-func LoadConfig(filePath string, fileName string) (*Config, error) {
+func LoadConfig() (*Config, error) {
+	v := viper.New()
+
+	pflag.StringP("config-name", "c" , FileName, "filename of the configuration to look for")
+	pflag.StringP("config-path", "p" , FilePath, "path where the configuration file is")
+
+	// Environment variables instead of v consistent with the config file
+	v.SetEnvPrefix("NRI_KUBERNETES")
+	v.AutomaticEnv()
```

Comment:
Do we want to support env for these flags?

Reply:
If could make the development easier or could change the location using envs instead of flags.

I would support everything just in case. it could be a NTH in the future to use remote configuration, for example.